css/src/core/bare-elements.scss

```diff
@@ -11,3 +11,15 @@ details > * {
 	// include a,li {} to override css reset/normalize; this is causing .focus-visible color issues
 	outline-color: inherit;
 }
+
+:where(a) {
+	color: $hyperlink;
+}
+
+:where(a:visited) {
+	color: $visited;
+}
+
+::part(visually-hidden) {
+	@include visually-hidden();
+}
```

Comment:
thinking this should go in /atomics/visually-hidden.scss so that when we eventually move .visually-hidden from docs to atlas they're combined into 
```scss
.visually-hidden,
::part(visually-hidden) {
	@include visually-hidden();
}
```

Reply:
could have it's own page and maybe we link to it from display.md, docs could be a followup work item

Reply:
Added to atomics with a comment to add the actual atomic class later